spec/classes/bird_spec.rb

```diff
@@ -198,6 +205,78 @@
           it { is_expected.to contain_yumrepo('bird') }
           it { is_expected.to contain_package('bird') }
         end
+        context 'with config_file_v4 and config_template_v4' do
```

Comment:
What about testing the positive case were the _content parameters are specified.